### DISCUSSION TITLE
Student Developer Pack Application + Copilot Student FAQs 🎒✨ (Updated)

### DISCUSSION THREAD

**Discussion by samus-aran** [community/community#111352] [GitHub Education]
Introducing our **FAQ** (Frequently Asked Questions) discussion designed specifically to help you breeze through the GitHub Education verification process with ease and confidence. We've listened to your concerns, sifted through the bucket of discussions, and compiled the most common ones into this one handy-dandy resource.

Remember to check out our [📌 Must-Read: Education Community Essential Guide – Updated Monthly!](https://gh.io/edu-comm-checkin) for all the latest in Education community. 📣 ⭐ 

> [!IMPORTANT]  
> **Benefits timing: Your academic benefits will be activated within 72 hours after successful verification.** 
>
> **New sign-ups for GitHub Copilot Student (and Copilot Pro, Pro+) plans are paused:** Pausing sign-ups allows us to serve existing customers more effectively. The pause affects the ability to activate Copilot Student or a Copilot paid plan. When signups resume, you will be able to activate the Education benefit if you are still eligible. In the meantime, Copilot Free remains [available](https://github.com/features/copilot/plans) for signups.
>
> We do not have an estimated timeframe to share yet for when the pause will end. Updates will be posted in this discussion post.
>

If you **don’t see your question**, **need a little extra guidance**, or want to **share any feedback** please feel free to [post it in the community](https://github.com/orgs/community/discussions/categories/github-education). We'll be keeping this **FAQ up-to-date** with the latest changes and additions to the verification process. 

So, bookmark the FAQ, check back often, and rest assured knowing that you'll always be equipped with the latest information to help you sail smoothly through verification.

<details id="quick-links">
<summary> Click Here for Quick Links 🔗🔗 👈 </summary>
<br />

- <a href="#how-to-edu">**How to get started** 🎒 🏃💨</a>
  - <a href="#who-is-eligible-for-the-pack"> Who is eligible for the pack? </a>
  - <a href="#where-find-the-verification-application">Where can I find the verification application? </a>
  - <a href="#why-do-i-need-to-submit-documentation"> Why do I need to submit documentation to get the GitHub Student Developer Pack?</a>
  - <a href="#what-types-of-evidence-does-the-student-developer-pack-application-accept"> What types of evidence does the Student Developer Pack application accept? </a>
  - <a href="#i-am-unable-to-submit-an-application-what-should-i-do"> I am unable to submit an application, what should I do? </a>
  - <a href="#how-long-after-i-submit-my-application-will-it-be-approved"> How long after I submit my application will it be approved? </a>
  - <a href="#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits"> How long after I’ve been approved will I receive my academic benefits? </a>
- <a href="#copilot-pro-faq-edu"> **Copilot Pro for Teachers & Copilot Student for Students** ✈️ 💻  </a>
  - <a href="#difference-between-copilot-free-and-copilot-for-education"> What is the difference between Copilot Free and Copilot for Students and Teachers? </a>
  - <a href="#copilot-free-for-students"> Is Copilot still free for verified students?</a>
  - <a href="#whats-change-copilot-for-students"> What is changing for Copilot for Students?</a>
  - <a href="#when-copilot-for-students-roll-out"> When will these Copilot Student changes roll out?</a>
  - <a href="#why-copilot-for-students"> Why are you making these changes?</a>
  - <a href="#self-select-model-picker"> Why can’t I self-select models in Copilot Student?</a>
  - <a href="#change-to-verified-educators-copilot"> Will this change affect verified GitHub Educators?</a>
  - <a href="#change-to-business-student-copilot"> Will this change affect Copilot Pro or Business/Enterprise customers?</a>
  - <a href="#get-involved-copilot-student"> How can I provide feedback or participate in preview programs?</a>
  - <a href="#tackling-copilot-fraud"> How are you preventing abuse and fraudulent GitHub Education usage?</a>
  - <a href="#enable-copilot-education"> How to enable Copilot Students for Students or Copilot Pro for Teachers? </a>
  - <a href="#transfer-copilot-free-to-pro"> I had Copilot Free and now have become a verified student or teacher. Do I need to do anything to transfer my Copilot Free plan?</a>
  - <a href="#verified-copilot-not-working"> I’m a verified student/teacher, but my Copilot is not working. What do I do?</a>
  - <a href="#copilot-coding-agent"> When will Copilot Coding Agent be available for  verified student/faculty accounts?</a>
- <a href="#edu-certification"> **Student GitHub Certification Benefit** 📃 🎟️  </a>
  - <a href="#how-do-i-redeem-edu-cert"> How do I redeem my GitHub Certification exam voucher as a student? </a>
  - <a href="#offer-isnt-available-end-term"> If you are seeing the message: "That offer isn't available right now, please try again later." when trying to claim a certification voucher </a>
  - <a href="#must-reads-edu-cert"> Why am I receiving an error when trying to redeem my voucher? </a>
- <a href="#common-reasons-tips">**Common Rejection Reasons + Tips** ❓💡 </a>
  - <a href="#i-dont-have-a-school-issued-email-address"> I don't have a school-issued email address, what other official proof of enrollment is accepted? </a>
  - <a href="#selected-school-incorrect-details"> My selected school states I must have a specific school-issued email address but it’s incorrect, what should I do?</a>
  - <a href="#double-sided-student-id-with-the-date-on-the-other-side"> I have a double-sided Student ID with the date on the other side. How do I submit this as evidence?</a>
  - <a href="#not-near-any-campus-location-for-selected-school"> You appear to not be near any campus location for the school you have selected. </a>
  - <a href="#selected-school-does-not-appear-to-have-a-campus-location-in-your-country"> The school you selected does not appear to have a campus location in your country. </a>
  - <a href="#document-does-not-appear-to-include-a-date"> Your document does not appear to include a date demonstrating current academic affiliation. </a>
  - <a href="#image-does-not-appear-to-contain-your-school-name"> The image you selected does not appear to contain your school name. </a>
  - <a href="#institution-doesnt-qualify"> My institution doesn’t qualify, but I want the Pack. What should I do? </a>
  - <a href="#outstanding-issues-with-your-account"> There are outstanding issues with your GitHub account. </a>
  - <a href="#last-verified-on-blank"> You were last verified as a student on "blank". </a>
  - <a href="#null-last-verified-on-date"> You were last verified as a "null" on... </a>
  - <a href="#location-error"> Error getting location. Try again?</a>
  - <a href="#use-front-camera"> I can only use my Front Camera to take a photo.</a>
  - <a href="#uploaded-proof-already-used"> Your uploaded proof has already been used previously.</a>
  - <a href="#translate-my-evidence"> I need to translate my evidence. What can I do? </a>
- <a href="#everything-else-edu">**Everything else** 📚 🤔 </a>
  - <a href="#application-rejected"> My application was rejected. What should I do?</a>
  - <a href="#application-keeps-getting-rejected"> My application keeps getting rejected. What should I do?</a>
  - <a href="#cant-find-my-question">I couldn’t find the answer I was looking for.</a>

</details>

<h2 id="how-to-edu">How to get started 🎒 🏃💨</h2>

<h3 id="who-is-eligible-for-the-pack">Who is eligible for the pack?</h3>

If you’re a **student aged 13+** and **enrolled in a degree or diploma-granting course of study**, the GitHub Student Developer Pack is for you. 👍 

When you apply, you will need to demonstrate current enrollment. The application process is individually tailored and the application interface and emails will guide you on the best next steps toward approval. For more information about the application process and what documents we accept, please read our [Education documentation](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student). You can apply by visiting [GitHub Education benefits](https://education.github.com/benefits).

<h3 id="where-find-the-verification-application">Where can I find the verification application? </h3>

The verification process has moved. To access it, log in to your GitHub account, click your profile icon in the top-right corner, and select **Settings**. In the left sidebar, go to **Billing and plans**, then click on **Plans and usage**. There, you’ll find a section labeled “**GitHub Education**”, where you can view your subscription details and manage your student or teacher verification.

Or go directly to the page here: https://github.com/settings/billing/summary#education-benefits

<h3 id="why-do-i-need-to-submit-documentation">Why do I need to submit documentation to get the GitHub Student Developer Pack?</h3>

We want to make it as simple as possible for students anywhere to easily apply and go through the GitHub Student Developer Pack application process, while also keeping our commitment to safeguard our partners and ourselves from bad actors who make our mission difficult.

<h3 id="what-types-of-evidence-does-the-student-developer-pack-application-accept">What types of evidence does the Student Developer Pack application accept?</h3>

If your school provides academic email to students, you will need to add it to your GitHub account and verify it.

In addition, we accept the following proof types, in order of preference:

1. Student ID
2. Transcript
3. Enrollment or affiliation letter
4. Class schedule

Your first and **last name, complete school name**, and a **date** demonstrating **current academic** affiliation _must_ appear in your document.

Make sure the image is clear and easy to read; if it looks blurry, please take a new photo. 👈 

If you don’t have an academic email and the application is requesting you to provide one associated with your selected school please see question <a href="#selected-school-incorrect-details"> My selected school states I must have a specific school-issued email address but it’s incorrect, what should I do?</a> for additional information.

If you don't have a student ID, or it doesn't include a date, you can upload additional accepted proof with a date that demonstrates your current enrollment. You may capture additional and multiple academic affiliation documents in the same image in order to fulfill all requirements.

If you have been repetitively unsuccessful, please check out our common reasons that applications for GitHub Global Campus are not approved.

<h3 id="i-am-unable-to-submit-an-application-what-should-i-do">I am unable to submit an application, what should I do?</h3>

If your application is **not proceeding to submission**, please ensure that you are providing sufficient, legible proof of current academic enrollment. You may take a photo of your academic proof directly with your device camera, _instead of using an upload_.

Please ensure that your browser settings **allow location verification**. You may alternatively use your phone if your computer settings do not allow this.

To troubleshoot the location check further, you may wish to try disabling your add-ons and/or clearing your cookies and cache, or try another browser. 💡 

If there is still any difficulty, raise a discussion on the [**GitHub Education community**](https://github.com/orgs/community/discussions/categories/github-education) and provide details and a screenshot of your issue. 👍 

<h3 id="how-long-after-i-submit-my-application-will-it-be-approved">How long after I submit my application will it be approved?</h3>

Applications are generally processed within a few days but **may take longer in peak periods such as the start of a new semester**. 

Please note our Community and Support team is unable to expedite the review process, so opening a support ticket will not decrease the review time.

<h3 id="how-long-after-ive-been-approved-will-i-receive-my-academic-benefits">How long after I’ve been approved will I receive my academic benefits?</h3>

**Congrats, your request has been approved and your benefits will be available soon!**

First off, congratulations on being approved. 🎉 When you submit your application, you'll have your academic affiliation verified and gain access to [GitHub Education](https://education.github.com/globalcampus/student) right away.

We aim to provide you with your educational benefits within 3 days of this approval status. However, during busy periods like the start of a new semester, it may take a bit longer.

If you have not received your education benefits within 5 days of your approval status please raise a discussion on the [GitHub Education community](https://github.com/orgs/community/discussions/categories/github-education) and the Education community team will be able to assist in resolving this for you. 🌟

You’ll get an email notification once your academic benefits are ready, or you can check your status anytime under “GitHub Education” in [Billing and plans](https://github.com/settings/billing/summary#education-benefits).. Meanwhile, as a verified GitHub Education student, you can explore the platform, connect with a local[ Campus Expert ](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from[ GitHub Community Exchange ](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange)and much more!

<h2 id="copilot-pro-faq-edu">Copilot Pro for Teachers & Copilot Student for Students ✈️ 💻</h2>

<h3 id="difference-between-copilot-free-and-copilot-for-education"> What is the difference between Copilot Free, Copilot Student for Students and Copilot Pro fo Teachers? </h3>

What is the difference between Copilot Free, Copilot Student for Students, and Copilot Pro for Teachers?
Copilot Free is available to all users, offering limited access to selected Copilot features.

As of **March 2026**, verified students receive access under the new GitHub Copilot Student plan, which includes enhanced benefits over Copilot Free but does not provide the full feature set of Copilot Pro. This dedicated student plan is tailored to learning-first scenarios and is separate from the benefits offered to teachers.

Verified educators (teachers and faculty) continue to receive Copilot Pro as part of their academic benefits through GitHub Education.

To become verified and access your respective benefits, visit the “GitHub Education” section under Billing and plans in your GitHub settings.

For a full overview of GitHub Copilot plans, [please refer to the documentation.](https://github.com/features/copilot/plans )

<h3 id="copilot-free-for-students">Is Copilot still free for verified students?</h3>

Yes. Copilot remains free for verified students as part of GitHub Education. Today, more than two million students use Copilot to learn and build, and we’re introducing a dedicated Copilot Student plan so the experience can evolve specifically for student needs.   

<h3 id="whats-change-copilot-for-students">What is changing for Students?</h3>

For verified students, your Copilot access will be managed under a new GitHub Copilot Student plan, alongside your existing GitHub Education benefits. The Copilot Student plan includes additional benefits over the Copilot Free plan but will not include the same level of access as the Copilot Pro plan. The Student Plan includes a change to premium model access, but the existing entitlement levels remain unchanged.

<h3 id="when-copilot-for-students-roll-out">When will these Copilot Student changes roll out?</h3>

Beginning **March 12, 2026**, verified students will be using a new GitHub Copilot Student plan when they access GitHub Copilot. Over the next several months, there may be more changes rolling out including rate limiting, abuse mitigation controls, and more changes to model access to ensure the sustainability of this student benefit. 

Over the next several months, we'll work closely with students and educators to find the right balance—one that supports how you learn today and scales to meet future needs. We'll share updates along the way and encourage you to share feedback in our Community Discussion so we can keep improving the experience together.   

<h3 id="why-copilot-for-students">Why are you making these changes?</h3>

GitHub Education serves a rapidly growing global community, having reached **over 11M students and teachers worldwide**. Use of AI tools in learning has scaled significantly. Over the past two years, we’ve listened closely to feedback from students and educators and learned a great deal about how Copilot is used in academic settings—from students who sometimes turn Copilot off to deepen their learning, to the unique needs and workflows that differ from professional development environments. These insights, along with the community’s continued growth, make this the right time to be thoughtful about evolving the Copilot student experience.  

These changes help us:  

* Keep Copilot free and sustainable for students long term  
* Ensure the experience supports learning-first scenarios  
* Improve performance and reliability as adoption grows  
* Continue investing in capabilities tailored for education  

Our goal is to balance broad access with long-term sustainability while delivering a high-quality experience designed for how students learn and build.  

<h3 id="self-select-model-picker">Why can’t I self-select models in Copilot Student?</h3>

Copilot Free and Student plans will now use Copilot **auto** model selection as the default and only model selection experience. Auto dynamically selects the best model for each task, removing the need for manual choice. Auto provides access to models across multiple model families, subject to plan restrictions. For more information, see [our documentation on supported models in auto mode](https://docs.github.com/copilot/reference/ai-models/supported-models#supported-ai-models-in-auto-model-selection).

We know this can be disappointing, but we’re making this change so we can keep Copilot free and accessible for millions of students around the world.

<h3 id="upgrade-to-copilot-pro">Can I upgrade my Copilot plan?</h3>

If you wish to upgrade from your GitHub Student plan to GitHub Copilot Pro, go to your Licensing page and click, **[Upgrade Copilot](https://github.com/settings/billing/licensing)**.

<h3 id="change-to-verified-educators-copilot">Will this change affect verified GitHub Educators?</h3>

No, the new Copilot Student plan was developed specifically for students. GitHub verified educators will continue to receive free access to Copilot Pro as part of their GitHub Education benefits.  

<h3 id="change-to-business-student-copilot">Will this change affect Copilot Pro or Business/Enterprise customers?</h3>

No. We’re introducing this plan specifically for students. Copilot Pro, Business, and Enterprise plans including pricing, features, and limits are not affected.  

<h3 id="get-involved-copilot-student">How can I provide feedback or participate in preview programs?</h3>

We welcome feedback from students and educators. You can:

* Share feedback in the [Community discussion](https://gh.io/copilot4edu) 👈 
* Connect with GitHub Education programs like Campus Experts
* Participate in targeted previews or experiments when invited

Your input helps shape how Copilot evolves for teaching and learning.

<h3 id="tackling-copilot-fraud">How are you preventing abuse and fraudulent GitHub Education usage?</h3>

We’re strengthening safeguards to ensure benefits go to legitimate students and educators, including:  

* Improved verification checks
* Continued monitoring for abnormal usage patterns
* Deactivation of accounts that violate program policies

These measures help keep the program sustainable while maintaining a positive experience for the community.

<h3 id="enable-copilot-education">How to enable Copilot Students for Students or Copilot Pro for Teachers?</h3>

Once you have been successfully approved as a verified student, your academic benefits (including GitHub Copilot Student) will become available within a few days. You will receive an email when your benefits are ready to be redeemed.

Students can claim Copilot for Students via the [benefits page](https://education.github.com/) or the dedicated [student signup link](https://github.com/github-copilot/free_signup).

Teachers and faculty, once verified, can claim Copilot Pro through their benefits page or the appropriate signup link for educators.

<h3 id="transfer-copilot-free-to-pro">I had Copilot Free and now have become a verified student or teacher. Do I need to do anything to transfer my Copilot Free plan?</h3>

Yes. If you are a newly approved student, you will need to enable GitHub Copilot Student by visiting the [GitHub Copilot Student sign up link](https://github.com/github-copilot/free_signup).

If you are a newly approved teacher, enable Copilot Pro via the educator benefits page.

You may need to close VS Code and reopen it after making the change.

If you encounter any issues enabling your new Copilot plan, please raise a discussion on the GitHub Education community or contact GitHub Education Support.

<h3 id="verified-copilot-not-working">I’m a verified student or teacher, but my Copilot is not working. What do I do?</h3>

If your GitHub Copilot isn't working, try these troubleshooting steps:

1. **Confirm you are verified:** Visit your application page to check for an **Approved** status. If your status is **green** 🟢, academic benefits should be available within a few days. If not, you may need to [re-verify yourself](https://github.com/settings/education/benefits).
2. **Check if your academic benefits are ready:** For students, look for a **purple** 🟣 **Approved** status. For teachers, ensure your educator benefits are active.
3. **Check if the correct Copilot plan is enabled:** Students should claim GitHub Copilot Student via the benefits page or [student signup link](https://github.com/github-copilot/free_signup). Teachers should claim Copilot Pro via their respective benefits page.
4. Consult the [Copilot FAQ](https://github.com/orgs/community/discussions/144674) for further assistance.

<h3 id="copilot-coding-agent">When will Copilot Coding Agent be available for verified student/faculty accounts?</h3>

Copilot Coding Agent is available to all verified GitHub Education students and educators.

<h2 id="edu-certification"> Student GitHub Certification Benefit  📃 🎟️  </h2>

<h3 id="how-do-i-redeem-edu-cert">How do I redeem my GitHub Certification exam voucher as a student?</h3>

To redeem your GitHub Certification exam voucher as a student, follow the detailed instructions in [this discussion post](https://github.com/orgs/community/discussions/165477), which covers eligibility, how to claim your voucher from the Student Developer Pack, and how to schedule your exam through GitHub Learn and Microsoft Learn.

<h3 id="offer-isnt-available-end-term">If you are seeing the message: "That offer isn't available right now, please try again later." when trying to claim a certification voucher</h3>

Due to constraints in the new system, GitHub is able to provide only a limited number of certification exam vouchers per term (e.g., fall, spring).  

If you see this message when attempting to claim a voucher through the Student Developer Pack, it means all available vouchers for the current term have already been claimed by other students.

**What you can do:**  
Another batch of vouchers will be released in January (or at the start of the next term).  If you have already redeemed your voucher and/or scheduled your exam, your access will not be disrupted. In the meantime, you can explore all free resources and exam preparation materials available via [GitHub Learn](https://learn.github.com/).
 
**Tip:** Keep an eye on announcements in the GitHub Education community for updates on voucher availability.

<h3 id="must-reads-edu-cert">Why am I receiving an error when trying to redeem my voucher?</h3>

**Important things to check if you are having trouble redeeming your voucher:**

**Voucher availability:**  
Vouchers are only available in limited quantities each term. If you no longer see a voucher code or receive a message such as "That offer isn't available right now, please try again later," all vouchers for the current term have been claimed. The next batch is typically released at the start of the next term (e.g., January).

**Eligibility:**  
You must be a currently verified student or teacher with access to the GitHub Student Developer Pack.  
If your verification has lapsed or you are not recognized as eligible, you will not be able to claim or redeem a voucher.

**Voucher validity:**  
Each voucher is for **single use** and for **one exam** (either Foundations OR Copilot). If you try to use it for both or attempt to reuse a code, you will see an error.

**Voucher expiration:**  
Each voucher has an expiration date (e.g., for 2025-2026, it is June 30, 2026). Expired vouchers cannot be redeemed.

**Rescheduling/cancellation:**  
Exam appointments must be rescheduled or cancelled **at least 24 hours** in advance. Otherwise, you risk forfeiting your voucher or exam fee.

**Platform transition:**  
GitHub certifications are now managed through Microsoft Learn and Pearson VUE. Make sure you are following the latest instructions and using the correct profiles on both platforms (see the [official discussion post for step-by-step guidance](https://github.com/orgs/community/discussions/165477)).

If you have verified all of the above and still cannot redeem your voucher, contact GitHub Education support for further assistance.

<h2 id="common-reasons-tips">Common Rejection Reasons + Tips ❓💡 </h2>

<h3 id="i-dont-have-a-school-issued-email-address">I don't have a school-issued email address, what other official proof of enrollment is accepted?</h3>

If your school provides academic email to students, you will need to add it to your GitHub account and verify it. ✅ 

If your school does not provide academic email to students, you will need to provide accepted proof of current enrollment.

For more information about the application process and what documents we accept, please read our [Education documentation](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student).

<h3 id="selected-school-incorrect-details">My selected school states I must have a specific school-issued email address but it’s incorrect, what should I do?</h3>

Some schools may require students to be verified by an already confirmed domain distributed by the school. If you are attending one of the schools that request specific verified academic emails and believe the school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education) using the dropdown "**My selected school has incorrect or incomplete information e.g. domains or campus location**".

You should **provide evidence** to support if your school has _additional academic domains_, has _changed its academic email issuing policy_, or _does not provide academic emails_. Reach out to your institution's IT team for this. 

Accepted types of evidence include a link to a publicly-accessible page on the school website or an affidavit from your school on school letterhead, detailing their current academic email issuing policy, or listing their academic domain(s), as it is relevant in your case. 👈 

_Please note that screenshots of emails purportedly sent by the school to non-academic webmail addresses are not considered to be reliable supporting documentation._

<h3 id="double-sided-student-id-with-the-date-on-the-other-side">I have a double-sided Student ID with the date on the other side. How do I submit this as evidence?</h3>

We recommend that you determine the best logistics to follow next steps based on the requirements.

In order to qualify for the Github education program, we must be able to verify that you are currently and actively enrolled in your academic program. Dated documents that will help you to be most successful in getting approved are the following:

- Student ID
- Transcript
- Enrollment or affiliation letter
- Class schedule

For countries utilizing non-standard academic calendars, you may need to capture the original document beside one with a converted date. 📆 

You may include multiple documents in your image, so long as they are legible. So as an example you could take a photo of your student ID card alongside a dated class schedule.

<h3 id="not-near-any-campus-location-for-selected-school">You appear to not be near any campus location for the school you have selected.</h3>

In order to qualify for the GitHub education program, we must be able to verify that you are either located close to your campus, or that you are a part of a verifiable distance learning program.

- You will have the most success applying if you are able to do so from your school’s campus.
- When asked, please enable the application to discover your location from the browser pop-up. Since settings for every browser are different, if you are having difficulty enabling us to see your location you should seek assistance from your browser's support pages. Information on browsers supported by GitHub can be found [here](https://docs.github.com/en/get-started/using-github/supported-browsers).
- If you are still having difficulties enabling your location check out[  this discussion on assessing your hardware or operating system's settings. ](https://github.com/orgs/community/discussions/42246)
- If you are using a VPN, please disable it when you reapply. If you are not using a VPN, try reapplying when connected to the internet from a different network.
- If you are a distance learning student, please attach official, dated proof indicating that your program is a distance learning program or you are a distance learning student when you reapply.

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education) using the dropdown "**My selected school has incorrect or incomplete information e.g. domains or campus location**". 👍 

You should **provide evidence** to support if your **school campus’s location has changed**, or an **additional campus needs to be associated** with it. 🏫 

<h3 id="selected-school-does-not-appear-to-have-a-campus-location-in-your-country">The school you selected does not appear to have a campus location in your country.</h3>

Please visit the "<a href="#not-near-any-campus-location-for-selected-school">You appear to not be near any campus location for the school you have selected</a>" section for guidance.

<h3 id="document-does-not-appear-to-include-a-date">Your document does not appear to include a date demonstrating current academic affiliation.</h3>

In order to qualify for the Github education program, we must be able to verify that you are currently and actively enrolled in your academic program. Legible, currently dated documents that will most help you to be successful in getting approved are the following:

- School ID
- Transcript
- Enrollment / affiliation verification letter
- Class schedule

For countries utilizing non-standard academic calendars, you may need to capture the original document beside one with a converted date. You may include multiple documents in your image, so long as they are legible.

<h3 id="image-does-not-appear-to-contain-your-school-name">The image you selected does not appear to contain your school name.</h3>

In order to qualify for the GitHub education program, we must be able to verify that you are enrolled in your school. That means, we need to clearly see your school name on your documentation. Here are a few tips we suggest:

- You may include multiple documents in your image. We look at all of the documents you submit collectively.
- Make certain that your documents are clear in your image, this may mean that you may need to print the document out and take a picture of it.
- Ensure that the school’s name you selected matches exactly on your submitted evidence. Text that is non-English should be translated.
- Logos are not sufficient.

<h3 id="institution-doesnt-qualify">My institution doesn’t qualify, but I want the Pack. What should I do?</h3>

We would love to provide all learners access to the real-world development tools provided by GitHub Education's partners in the future, but right now to become a member of the Pack you must be 13+ years of age and currently enrolled in an accredited degree or diploma granting course of study.

You are welcome to ask your instructor at your code school or boot camp to apply to become a [GitHub Education partner](https://education.github.com/partners/schools).

If approved, program participants will become eligible to receive all of the benefits of GitHub Education.

<h3 id="outstanding-issues-with-your-account">There are outstanding issues with your GitHub account</h3>

If you are seeing this message when applying for the Student Developer Pack and can not proceed please contact [Account or billing issues](https://support.github.com/request) and they will explain the details of this message.

This message can be an indication of many things that only the Support team can answer and resolve. We thank you for your patience and understanding whilst we are ensuring that bad actors impersonating students don't jeopardize our ability to provide academic benefits of value.

<h3 id="last-verified-on-blank">You were last verified as a student on "blank"</h3>

If you are seeing this message when you are trying to reverify we would request that you raise a discussion on the [**GitHub Education community**](https://github.com/orgs/community/discussions/categories/github-education), provide details and a screenshot of your issue and the Education community team will be able to assist in resolving this for you. 🌟

<h3 id="null-last-verified-on-date"> You were last verified as a "null" on...</h3>

If you are seeing this message this usually indicates that your application has been Approved and is still within its 72 hours wait period before receiving your academic benefits. The "null" will be updated when you receive your academic benefits to reflect the type of application you had submitted. You do not need to do anything else at this time.

You’ll get an email notification once your academic benefits are ready, or you can check your status anytime under “GitHub Education” in [Billing and plans](https://github.com/settings/billing/summary#education-benefits). Meanwhile, as a verified[ GitHub Education ](https://education.github.com/globalcampus/student)student, you can explore the platform, connect with a local[ Campus Expert ](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from[ GitHub Community Exchange ](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange)and much more!

<h3 id="location-error"> Error getting location. Try again?</h3>

To troubleshoot **"Error getting location. Try again?"** please try the following steps:

* Please configure your browser and operating system to allow location access. You may not use a VPN when applying for these benefits.
* Please try disabling your add-ons and/or clearing your cookies and cache, or try multiple different browsers. If possible, we recommend using the Chrome browser.
* Please also check that your platform settings allow your browser to provide location information.
* Please remember to log out and log back in after making the above adjustments before re-applying.

If you continue to have difficulty, we recommend applying from your smartphone using your mobile network and not the same WiFi that you used on your computer.

<h3 id="use-front-camera">I can only use my Front Camera to take a photo</h3>

We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a phone to capture.

1. Please ensure that you have allowed camera access on the browser you're using to apply. The steps for this will vary depending on which browser you are using. Information on browsers supported by GitHub can be found [here](https://docs.github.com/en/get-started/using-github/supported-browsers).
2. If you can, print off your evidence and take a photo of this copy.
3. Ensure your image is legible. If you are unable to read the text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone.
4. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.
6. Alternatively you may use another mobile device or computer.

<h3 id="uploaded-proof-already-used">Your uploaded proof has already been used previously</h3>

It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students to **submit a live photo of their evidence rather than an upload**.

This rejection reason usually indicates that an image has been previously used. We recommend that you supply a **new live capture** of your evidence upon re-applying to avoid this.

<h3 id="translate-my-evidence">I need to translate my evidence. What can I do?</h3>

Ensure your _billing information_ and _identification on your academic evidence_ are **accurate**. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

We are able to translate a number of languages but if you feel that the system is unable to process your particular language then we would advise you to take a photo of the original piece of evidence next to a typed English translation, without using special characters / accent marks. Your name on the account may need to match the translated proof as well.

Ensure your image is legible. If you are unable to read the text on the image itself then the verification process may not be able to either.

<h2 id="everything-else-edu">Everything else... 📚 🤔  </h2>

<h3 id="application-rejected">My application was rejected. What should I do?</h3>

Please double check that you have updated your [billing information](https://github.com/settings/billing/payment_information) and [two-factor authentication](https://github.com/settings/security), and addressed any other rejection reasons.

Also check out the education documentation on the [common reasons an application is rejected](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved) and how to correct it.

<h3 id="application-keeps-getting-rejected">My application keeps getting rejected. What should I do?</h3>

Only re-apply if you have made any updates to your profile, evidence or application after a failed submission. Please double check that you have updated your [billing information](https://github.com/settings/billing/payment_information) and [two-factor authentication](https://github.com/settings/security), and addressed any other rejection reasons.

<h3 id="cant-find-my-question">I couldn’t find the answer I was looking for</h3>

We continuously update these FAQs but if you are unable to find the answer to your question please raise a discussion on the [GitHub Education community](https://github.com/orgs/community/discussions/categories/github-education) and the Community team will be happy to assist 🌟

**Comment by admin** [comment]
As of June 1, 2026: Usage-based billing for GitHub Copilot is now live for all users and Copilot code review consumes GitHub Actions minutes, in addition to GitHub AI Credits.  

As part of this release, we're also launching new user-level budget controls, expanded context windows, and enabling upgrades to Copilot Max. 

⭐ For the most up-to-date information, we have provided more details and a FAQ in [this discussion](https://github.com/orgs/community/discussions/197089).
